apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -326,7 +343,7 @@ def render(element_html, data):
     return html
 
 
-def get_format_string(is_complex=False, allow_fractions=False, message=None):
+def get_format_string(is_complex=False, allow_fractions=False, message=None) -> str:
```

Comment:
Missing type annotations on input arguments. The type goes before the default argument, send me a DM if you have trouble finding examples.

Reply:
Fixed. Note to self if I dig this up again: https://docs.python.org/3/library/typing.html#typing.Optional and https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#functions

Reply:
Also, since the argument `message` is of type `str`, the type checker is throwing an error because the default `None` is not a `str`. Should I change the default to an empty string instead?